src/index.js

```diff
@@ -10,7 +11,8 @@ const SortableList = SortableContainer(({ forwardRef, className, style, children
  
 const GridTable = props => {
 
-    const tableManager = useTableManager(props);
+    const tableId = props.tableId === '' ? nanoid(10) : props.tableId;
```

Comment:
Wouldn't that generate a new id every render? Also, it would be better to put it in the useTableManger hook directly as this is where all initializations are happening already.

Reply:
Yes, new id will be generated each render, or you can pass custom id to each table, if you want to keep it same. I moved it to the tableManager.